fhir-persistence-params/src/main/java/com/ibm/fhir/persistence/params/database/PlainDerbyParamValueProcessor.java

```diff
@@ -0,0 +1,327 @@
+/*
+ * (C) Copyright IBM Corp. 2022
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+ 
+package com.ibm.fhir.persistence.params.database;
+
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+import com.ibm.fhir.database.utils.derby.DerbyTranslator;
+import com.ibm.fhir.persistence.exception.FHIRPersistenceException;
+import com.ibm.fhir.persistence.params.api.IParameterIdentityCache;
+import com.ibm.fhir.persistence.params.api.ParamSchemaConstants;
+import com.ibm.fhir.persistence.params.api.ParameterNameDAO;
+import com.ibm.fhir.persistence.params.model.CodeSystemValue;
+import com.ibm.fhir.persistence.params.model.CommonCanonicalValue;
+import com.ibm.fhir.persistence.params.model.CommonCanonicalValueKey;
+import com.ibm.fhir.persistence.params.model.CommonTokenValue;
+import com.ibm.fhir.persistence.params.model.CommonTokenValueKey;
+import com.ibm.fhir.persistence.params.model.LogicalResourceIdentKey;
+import com.ibm.fhir.persistence.params.model.LogicalResourceIdentValue;
+
+/**
+ * Derby variant of the plain schema message handler which is needed because Derby
```

Comment:
"message handler" sounds a bit out of place here.  maybe change that to "value processor"?